helpers/enable-j5-activate.sh

```diff
@@ -78,5 +78,38 @@ j5activate() {
             return 1
         fi
     fi
+
+  complete -F _j5switch j5switch
+  complete -F _j5setup j5setup
+
+
+}
+_j5setup()
+{
+    local cur=${COMP_WORDS[COMP_CWORD]}
+    COMPREPLY=( $(compgen -W "$(echo "--no-ui --skip-build --help --skip-docs --with-ui --locale --copy-from-custom-staging-dir --no-service-startup --no-db-migration --no-schema-management --admin-password --ignore-installed-apps-requirement --dry-run" | tr " " "\n")" -- $cur ))
+}
+
+_j5switch()
+{
+    local cur=${COMP_WORDS[COMP_CWORD]}
+    if [ $COMP_CWORD -eq 1 ]
+        then
+            COMPREPLY=( $(compgen -W "$(find ~/work/git/ -maxdepth 1 -type d -printf "%f\n")" -- $cur ))
```

Comment:
Hard coding `~/work/git` doesn't feel right. Surely we can do something like $(dirname $(dirname $(dirname ... $0 ...))) :P

Reply:
No this is hard coded in more important places so it's pointless to try and be smart

Reply:
I thought so. I think the scripts are a bit more forgiving on windows.

Reply:
ok made it better anyways